ephios/user_management/templates/user_management/userprofile_form.html

```diff
@@ -93,6 +93,11 @@ <h3 class="card-title">{% translate "Qualifications" %}</h3>
             <a role="button" class="btn btn-secondary"
                href="{% url "user_management:userprofile_list" %}">{% trans "Cancel" %}</a>
             <button type="submit" class="btn btn-primary float-right">{% trans "Save" %}</button>
+            {% if userprofile %}
```

Comment:
please show this to user with the delete_userprofile permission only. and please add some margin to the save button, they stick to each other rn. would also prefer the delete button on the left side (to the right of the cancel button) so that the destructive actions are seperated from the constructive actions. (same as in shift_form)